components/compositing/compositor_thread.rs

```diff
@@ -74,6 +110,37 @@ impl RenderListener for CompositorProxy {
     }
 }
 
+pub enum EmbedderMsg {
```

Comment:
I'm assuming the `WindowMethod`(and `WindowEvent`), are a way to offer a consistent api to the embedder, and that we would keep them. Do you mean that `EmbedderMsg` isn't an appropriate name because they are handled by `Servo`?